doc/api_assets/api.js

```diff
@@ -28,16 +29,17 @@
       if (mq.matches) {
         document.documentElement.classList.add('dark-mode');
       }
-    } else if (userSettings === 'true') {
+    } else if (storedTheme === 'dark') {
       document.documentElement.classList.add('dark-mode');
     }
 
     if (themeToggleButton) {
       themeToggleButton.hidden = false;
       themeToggleButton.addEventListener('click', function() {
-        sessionStorage.setItem(
-          kCustomPreference,
-          document.documentElement.classList.toggle('dark-mode'),
+        const isDark = document.documentElement.classList.toggle('dark-mode');
```

Comment:
is a variable needed?

Reply:
i suspect it makes the ternary easier to read

Reply:
But do we care about that? On a legacy file?

Reply:
np, will skip the variable